tools/readme_generator/make_readme.py

```diff
@@ -49,17 +49,19 @@ def generate_READMEs(app_path: Path):
         )
         return
 
-    env = Environment(loader=FileSystemLoader(Path(__file__).parent / "templates"))
+    env = Environment(loader=FileSystemLoader(README_GEN_DIR / "templates"))
+
+    screenshots: List[str] = []
 
-    screenshots: List[str]
-    screenshots = []
-    if (app_path / "doc" / "screenshots").exists():
+    screenshots_dir = app_path / "doc" / "screenshots"
+    if screenshots_dir.exists():
         # only pick files (no folder) on the root of 'screenshots'
-        for entry in os.scandir(os.path.join(app_path, "doc", "screenshots")):
-            if os.DirEntry.is_file(entry):
-                # ignore '.gitkeep' or any file whose name begins with a dot
```

Comment:
i think this comment is important for clarity

Reply:
woops yeah okay, adding it back.

Reply:
in general, we should comment on our code better btw 😅